blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubEnterpriseOrganization.java

```diff
@@ -0,0 +1,44 @@
+package io.jenkins.blueocean.blueocean_github_pipeline;
+
+import io.jenkins.blueocean.rest.hal.Link;
+import io.jenkins.blueocean.rest.impl.pipeline.scm.ScmOrganization;
+import io.jenkins.blueocean.rest.impl.pipeline.scm.ScmRepositoryContainer;
+
+/**
+ * @author cliffmeyers
+ */
+public class GithubEnterpriseOrganization extends ScmOrganization {
```

Comment:
We need similar thing for bitbucket. May be create a non final class `ScmOrganizationWrapper`, move it to scm package in blueocean-pipeline-impl module and then provide implementation as `GithubEnterpriseOrganization extends ScmOrganizationWrapper` where it overrides `getAvatar()` and returns null? This will allow bitbucket case also to simply override getAvatar() to serve null.